UserService/app/src/main/resources/application.properties

```diff
@@ -1,2 +1,13 @@
 spring.application.name=user-service
```

Comment:
Let us configure properties/yaml files always with profiles - local,dev,prod etc. 
This way it helps to manage properties in case we need to deploy application different infra environments.